src/app.scss

```
@@ -240,3 +240,19 @@
 .pf-v5-c-menu-toggle {
     padding-inline: var(--pf-v5-global--spacer--md) calc(var(--pf-v5-global--spacer--md) * 0.75);
 }
+
+// Toolbar actions styling
+.header-toolbar {
+    display: flex;
+    flex-flow: row wrap;
+    gap: var(--pf-v5-global--spacer--sm);
+
+    .files-search {
+        flex: 2;
```

```suggestion
        flex: 15ch;
```

If we change flex here to a value, like 15ch, then it's shorthand for `1 1 15ch`, which means grow: yes, shrink: yes, and try to keep it around 15ch unless there's enough space for more.

This means it'll wrap and not truncate the placeholder text, at least in English. It does actually start to wrap when there's the placeholer text plus a little space. Without specifying the width, it actually wraps a little too late and the default in-English placeholder gets truncated, and it's likely to truncate in most other languages too.

Yes, it's a magic number, and not something like `fit-content`, `min-content`, or `max-content` CSS keywords, but that's because it's an input which doesn't really abide by that (as the placeholder content is not really content, and the actual content is made to fluctuate). Most other elements do abide by fit, min, and max content, however.